thesis-api-v2/src/main/java/pl/edu/agh/quizzesthesis/business/service/DefinitionProcessingService.java

```diff
@@ -174,6 +174,27 @@ public DefinitionProcessing removeAnswerFromDefinition() {
             return new DefinitionProcessing(newDefinition, answer, articleTitle);
         }
 
+        public DefinitionProcessing removeTextInBracketsFromDefinition() {
```

Comment:
Since we are removing all text in brackets (with brackets included), we could remove `(` and `)` from `DELIMITER_PREFIXES` and `DELIMITER_POSTFIXES`.